src/OpenLoco/src/Objects/Object.cpp

```diff
@@ -650,4 +651,18 @@ namespace OpenLoco
         auto search = _vanillaObjects.find(std::make_tuple(getName(), getType()));
         return search != _vanillaObjects.end();
     }
+
+    // 0x00473BC7
+    void objectCreateIdentifierName(char* dst, const ObjectHeader& header)
+    {
+        for (auto& c : header.name)
+        {
+            if (c != ' ')
+            {
+                *dst++ = c;
+            }
+        }
```

Comment:
If its not possible to have something like "AA BBB" then it could break when it stumbles upon a whitespace.

Reply:
This is just taken from vanilla. I guess we could just change it to a trim.